### PR TITLE
Peer review edits from Alex

### PR DESCRIPTION
**Pull request by JoanneOFlynn2018**

### CODE REVIEW

File: doc-Service-Telemetry-Framework/modules/proc_deploying-smart-gateways.adoc [hunk 1]
```diff
@@ -57,15 +57,15 @@ do
 done
 ----
 
-. To modify the manifests, change the `metadata.name` field and `spec.amqpUrl` field to include the cloud identifier from your schema. For more information, see <<planning-amqp-address-prefixes_advanced-features>>. To view example Smart Gateway manifests, see <<example-manifests_advanced-features>>.
+. Create a new Smart Gateway manifest configuration file from the `/tmp/cloud1-smartgateways.yaml` file. Modify the `metadata.name` and `spec.amqpUrl` fields to include the cloud identifier from your schema. For more information, see <<planning-amqp-address-prefixes_advanced-features>>. To view example Smart Gateway manifests, see <<example-manifests_advanced-features>>.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_deploying-smart-gateways.adoc:60]: @leifmadsen This is the main point for you to review please Leif.

Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/proc_deploying-smart-gateways.adoc:60]:
```suggestion
. Modify the Smart Gateway manifest in the `/tmp/cloud1-smartgateways.yaml` file. Adjust the `metadata.name` and `spec.amqpUrl` fields to include the cloud identifier from your schema. For more information, see <<planning-amqp-address-prefixes_advanced-features>>. To view example Smart Gateway manifests, see <<example-manifests_advanced-features>>.
```

File: doc-Service-Telemetry-Framework/modules/con_configuring-multiple-clouds.adoc [hunk 2]
```diff
@@ -23,9 +23,9 @@
 [id="configuring-multiple-clouds_{context}"]
 = Configuring multiple clouds
 
-To configure multiple {OpenStack} clouds to target a single instance of {Project} ({ProjectShort}), complete the following tasks:
+You can configure multiple {OpenStack} clouds to target a single instance of {Project} ({ProjectShort})::
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/con_configuring-multiple-clouds.adoc:26]:
```suggestion
You can configure multiple {OpenStack} clouds to target a single instance of {Project} ({ProjectShort}):
```

File: doc-Service-Telemetry-Framework/modules/ref_example-manifests.adoc [hunk 3]
```diff
@@ -25,9 +25,9 @@
 
 IMPORTANT: The content in the following examples might be different to the file content in your deployment. Copy the manifests in your deployment.
 
-Ensure that the `name` and `amqp_url` parameters of each Smart Gateway match the names you want to use for your clouds. For more information, see <<planning-amqp-address-prefixes_advanced-features>>.
+Ensure that the `name` and `amqp_url` parameters of each Smart Gateway match the names that you want to use for your clouds. For more information, see <<planning-amqp-address-prefixes_advanced-features>>.
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/ref_example-manifests.adoc:28]:
```suggestion
Ensure that the `name` and `amqpUrl` parameters of each Smart Gateway match the names that you want to use for your clouds. For more information, see <<planning-amqp-address-prefixes_advanced-features>>.
```

File: doc-Service-Telemetry-Framework/modules/proc_deploying-smart-gateways.adoc (hunk 1 above)
Comment (JoanneOFlynn2018) [reply doc-Service-Telemetry-Framework/modules/proc_deploying-smart-gateways.adoc:60]: Ah ok, thanks. There was a bit of confusion and we were trying to be more explicit in the step. Your suggestion below looks perfect.

**Pull request merged**